ops/ansible/playbooks-ccs/build_bfd-db-migrator.yml

```diff
@@ -0,0 +1,52 @@
+---
+##
+# Configures the BFD DB Migrator Service for CCS env
+##
+- name: Configure DB Migrator Server
+  hosts: all
+  become: true
+  remote_user: ec2-user
+  gather_facts: yes
+  vars:
+    ansible_ssh_pipelining: no
+    env: "test"
+
+  tasks:
+    - name: Include global variables
+      include_vars:
+        file: vars/000_cross_env_vars.yml
+      tags: [pre-ami, post-ami]
+
+    - name: Include env specific variables
+      include_vars:
+        dir: vars/{{ env }}
+      tags: [pre-ami, post-ami]
+
+    - name: Apply base Cloudwatch Agent Role
+      import_role:
+        name: cloudwatch-agent-instrumented
+
+    - name: Apply latest host security patches
+      import_role:
+        name: base_patch
+
+    - name: Apply DB Migrator Role
+      import_role:
+        name: bfd-db-migrator
```

Comment:
Assuming we are expecting Keith's role-based stuff is a pre-requisite?

Reply:
The migrator is _effectively_ a prereq of Keith's proposed database role changes, but this _ansible role_ is orthogonal.